hack/ovn-kind-cno.sh

```diff
@@ -26,38 +35,30 @@ if ! command -v kubectl; then
   exit 1
 fi
 
-# Detect IP to use as API server
```

Comment:
The apiserver "autodectects" its external IP, and since we only have one interface in the nodes, there is no problem with "choosing" the wrong one